src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,8 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            System.out.println(new Lottery().getRandomBall());
```

Comment:
You create an object of the Lottery class in each iteration of the loop. Put it in a variable before the loop and use it in the loop